test/services/postcard.spec.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Sinon   = require('sinon');
+
+var Config = require('../../config');
+
+require('angular-mocks');
+
+describe('postcard service', function () {
+
+  var $q;
+  var $rootScope;
+  var API;
+  var Postcard;
+
+  beforeEach(Angular.mock.inject(function ($injector) {
+    $q         = $injector.get('$q');
+    $rootScope = $injector.get('$rootScope');
+    API        = $injector.get('API');
+    Postcard   = $injector.get('Postcard');
+  }));
+
+  describe('create', function () {
+
+    it('calls the correct endpoint and with correct params', function () {
+      var payload = { id: 'psc_id' };
+
+      Sinon.stub(API, 'post').returns();
```

Comment:
nit: for posterity, you should stub out the return value with something that is the same type as the original function. since `API.post` returns a promise, you should make the stub return a promise too:

```js
Sinon.stub(API, 'post').returns($q.resolve());
```

i can go over this irl if its a bit confusing